### PR TITLE
Add a contribution guide to the php-runtimes refactor

### PR DESCRIPTION
**Pull request by PineappleIOnic**
This PR will add a contributing.md file to the refactor branch of php-runtimes detailing how to contribute to the repository

### CODE REVIEW

**Comment by christyjacob4** [comment]
@PineappleIOnic  didn't we have another PR for this which was already reviewed and merged ?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Contributing
```
Comment (christyjacob4) [inline CONTRIBUTING.md:1]: Let's use the contributing guide we specify here as the base template and then add the specifics of this repo to that.
https://github.com/appwrite/appwrite/issues/2118

Comment (PineappleIOnic) [reply CONTRIBUTING.md:2]: Done 👍

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+```bash
+docker run --rm --interactive --tty --volume "%cd%":/app composer update --ignore-platform-reqs --optimize-autoloader --no-plugins --no-scripts --prefer-dist
+```
+
+4. Build all runtimes locally by running the following command within the project root:
+```
+./runtimes/buildLocalOnly.sh
+```
+
+3. [Follow our contribution guide to learn how you can add support for more runtimes.](https://github.com/appwrite/php-runtimes/blob/refactor/docs/adding-a-new-runtime.md)
```
Comment (lohanidamodar) [inline CONTRIBUTING.md:95]: Let's follow similar writing
https://github.com/utopia-php/storage/blob/cb7fb768475b7c57cc38bcb20095a4d7d30fbe04/CONTRIBUTING.md?plain=1#L77

**Review by lohanidamodar — CHANGES_REQUESTED** [review]
small change, rest looks good

**Comment by christyjacob4** [comment]
@PineappleIOnic  is this still relevant after we migrated the contributing guide over to open-runtimes?  
https://github.com/open-runtimes/open-runtimes/blob/main/docs/add-runtime.md

**Pull request closed without merging**